### PR TITLE
Fix broken example

### PR DESCRIPTION
**Pull request by lmagyar**
Issues:
- without start_listening(), subscribe_events() will never receive it's response from the server
- start_listening() have to be the first call, but it blocks, so we have to put it in a task, and we have to eg. log it's exception
- we need eg. a sleep to wait for the messages, but we could also await for the created task also

fixes: #186

### CODE REVIEW

File: example.py [hunk 1]
```diff
@@ -37,10 +37,16 @@ async def connect(args: argparse.Namespace, session: ClientSession) -> None:
...
-        await client.start_listening()
+        asyncio.create_task(log_exceptions(client.start_listening()))
+        await client.subscribe_events(log_events)
+        await asyncio.sleep(60)
 
+async def log_exceptions(awaitable):
+    try:
+        await awaitable
+    except Exception as e:
+        LOGGER.error(e)
```
Comment (marcelveldt) [inline example.py:49]: Exceptions need to be handled, not ignored.
Also if you want to log them, maybe a stack trace is convenient.
So a simple LOGGER.exception would be better, especially for unhandled exceptions

File: example.py [hunk 2]
```diff
@@ -37,10 +37,16 @@ async def connect(args: argparse.Namespace, session: ClientSession) -> None:
     """Connect to the server."""
     websocket_url = args.url.replace("http", "ws") + "/api/websocket"
     async with HomeAssistantClient(websocket_url, args.token, session) as client:
-        await client.subscribe_events(log_events)
         # start listening will wait forever until the connection is closed/lost
-        await client.start_listening()
+        asyncio.create_task(log_exceptions(client.start_listening()))
```
Comment (marcelveldt) [inline example.py:41]: listener_task = asyncio.create_task(client.start_listening())
await client.subscribe_events(log_events)
await listener_task

Comment (lmagyar) [reply example.py]: Yes, agree, that was what I meant in "we could also await for the created task also". :)

File: example.py (hunk 1 above)
Comment (lmagyar) [reply example.py]: The `await listener_task` is a much better solution, using that.

**Review by marcelveldt — APPROVED** [review]

**Comment by lmagyar** [comment]
Closing, as this got fixed with v1.2.0.

**Pull request closed without merging**